golem-cli/src/model/app.rs

```diff
@@ -1602,8 +1603,7 @@ mod app_builder {
                 }
             }
 
-            // TODO: once app model is moved to golem-cli then make this generated from the app command
-            // TODO: update this list based on current commands
+            // !TODO: once app model is moved to golem-cli then make this generated from the app command
```

Comment:
Didn't you want to do this in this PR?

Reply:
I can do it in this one or the next one, there are a few more similiar, just wanted to reach a state which works as before and not blocking others